src/test/resources/datasets/user/login_with_oauth2_init.yml

```diff
@@ -0,0 +1,222 @@
+advertisement:
+  - id: 65e3ee49-5927-40be-aafd-0461ce45f295
```

Comment:
I don't think you're using all the tables for a single test. Why don't you specify only those tables that are really used during the test scenarios?